buildenv/jenkins/openjdk_tests

```diff
@@ -165,8 +173,7 @@ if (JDK_VERSIONS.size() > 1 || JDK_IMPLS.size() > 1 || PLATFORMS.size() >1 || PL
         println "LABEL: ${LABEL}"
 
         stage('Queue') {
-            String[] onlineNodes = nodesByLabel(LABEL)
```

Comment:
`onlineNodes` is used in line 194. By removing it here, I think it will break line 194.

Reply:
how do mean @llxia it was moved to L297

Reply:
By moving to line 297, `onlineNodes` becomes a local variable in the function. However, we need to use `onlineNodes` in https://github.com/adoptium/aqa-tests/blob/ad669e153b314745f8a68aabecd640a1be4a264d/buildenv/jenkins/openjdk_tests#L194

Reply:
Yes, that is my bad, I did not see that it was being used below in this case for determining dynamic agents... so either we skip calling the new method here or change the new method to just return the array and do the check for less than zero in the 2 places we need to know the answer.

Reply:
hi @norbline - to resolve this, for now you can put 
`String[] onlineNodes = nodesByLabel(LABEL)` as a line between L193 and L194
https://github.com/adoptium/aqa-tests/blob/ad669e153b314745f8a68aabecd640a1be4a264d/buildenv/jenkins/openjdk_tests#L193-L194

Reply:
link to grinder https://ci.adoptopenjdk.net/job/Grinder/2795/